src/lakefs_spec/client_helpers.py

```diff
@@ -63,3 +65,21 @@ def revert(client: LakeFSClient, repository: str, branch: str, parent_number: in
     client.branches_api.revert_branch(
         repository=repository, branch=branch, revert_creation=revert_creation
     )
+
+
+def rev_parse(
+    client: LakeFSClient,
+    repository: str,
+    ref: str,
+    parent: int = 0,
+    prefixes: List[str] = [],
+    objects: List[str] = [],
```

Comment:
1) I would not support these arguments until they are necessary, and
2) do not use mutable containers (lists, dicts) as default arguments - either use `list()` or make it a `list[str] | None`.